bundle/manifests/watermarkpodautoscaler.clusterserviceversion.yaml

```diff
@@ -0,0 +1,189 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
you can test the render with this link: https://operatorhub.io/preview

Reply:
I did some boilerplate for the operator but I think taking this to the finish line will require more work (I will add a card, as part of the todo next)